2014-11-28 23:26:33.114 7214 INFO neutron.common.config [-] Logging enabled!
2014-11-28 23:26:33.472 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.488 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.496 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.505 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.513 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.522 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.529 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.538 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.545 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.555 7214 INFO oslo.messaging._drivers.impl_rabbit [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.575 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.589 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.594 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.608 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:26:33.638 7214 INFO neutron.plugins.openvswitch.agent.ovs_neutron_agent [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b None] Mapping physical network physnet-tenant to bridge br-tenant
2014-11-28 23:26:34.425 7214 INFO neutron.plugins.openvswitch.agent.ovs_neutron_agent [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b None] Agent initialized successfully, now running... 
2014-11-28 23:26:34.434 7214 INFO neutron.plugins.openvswitch.agent.ovs_neutron_agent [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b None] Agent out of sync with plugin!
2014-11-28 23:33:50.949 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] Failed to consume message from queue: Socket closed
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit Traceback (most recent call last):
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 655, in ensure
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit     return method()
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 735, in _consume
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.connection.drain_events(timeout=timeout)
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/connection.py", line 281, in drain_events
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.transport.drain_events(self.connection, **kwargs)
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/transport/pyamqp.py", line 94, in drain_events
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit     return connection.drain_events(**kwargs)
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 299, in drain_events
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit     chanmap, None, timeout=timeout,
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 362, in _wait_multiple
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit     channel, method_sig, args, content = read_timeout(timeout)
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 326, in read_timeout
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.method_reader.read_method()
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 189, in read_method
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit     raise m
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit IOError: Socket closed
2014-11-28 23:33:50.949 7214 TRACE oslo.messaging._drivers.impl_rabbit 
2014-11-28 23:33:50.952 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:50.952 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] Failed to consume message from queue: Socket closed
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit Traceback (most recent call last):
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 655, in ensure
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit     return method()
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 735, in _consume
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.connection.drain_events(timeout=timeout)
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/connection.py", line 281, in drain_events
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.transport.drain_events(self.connection, **kwargs)
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/transport/pyamqp.py", line 94, in drain_events
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit     return connection.drain_events(**kwargs)
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 299, in drain_events
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit     chanmap, None, timeout=timeout,
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 362, in _wait_multiple
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit     channel, method_sig, args, content = read_timeout(timeout)
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 326, in read_timeout
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.method_reader.read_method()
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 189, in read_method
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit     raise m
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit IOError: Socket closed
2014-11-28 23:33:50.952 7214 TRACE oslo.messaging._drivers.impl_rabbit 
2014-11-28 23:33:50.953 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:50.953 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] Failed to consume message from queue: Socket closed
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit Traceback (most recent call last):
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 655, in ensure
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit     return method()
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 735, in _consume
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.connection.drain_events(timeout=timeout)
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/connection.py", line 281, in drain_events
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.transport.drain_events(self.connection, **kwargs)
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/transport/pyamqp.py", line 94, in drain_events
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit     return connection.drain_events(**kwargs)
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 299, in drain_events
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit     chanmap, None, timeout=timeout,
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 362, in _wait_multiple
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit     channel, method_sig, args, content = read_timeout(timeout)
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 326, in read_timeout
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.method_reader.read_method()
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 189, in read_method
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit     raise m
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit IOError: Socket closed
2014-11-28 23:33:50.953 7214 TRACE oslo.messaging._drivers.impl_rabbit 
2014-11-28 23:33:50.954 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:50.955 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] Failed to consume message from queue: Socket closed
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit Traceback (most recent call last):
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 655, in ensure
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit     return method()
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 735, in _consume
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.connection.drain_events(timeout=timeout)
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/connection.py", line 281, in drain_events
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.transport.drain_events(self.connection, **kwargs)
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/transport/pyamqp.py", line 94, in drain_events
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit     return connection.drain_events(**kwargs)
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 299, in drain_events
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit     chanmap, None, timeout=timeout,
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 362, in _wait_multiple
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit     channel, method_sig, args, content = read_timeout(timeout)
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 326, in read_timeout
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.method_reader.read_method()
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 189, in read_method
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit     raise m
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit IOError: Socket closed
2014-11-28 23:33:50.955 7214 TRACE oslo.messaging._drivers.impl_rabbit 
2014-11-28 23:33:50.956 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:50.956 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] Failed to consume message from queue: Socket closed
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit Traceback (most recent call last):
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 655, in ensure
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit     return method()
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/impl_rabbit.py", line 735, in _consume
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.connection.drain_events(timeout=timeout)
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/connection.py", line 281, in drain_events
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.transport.drain_events(self.connection, **kwargs)
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/kombu/transport/pyamqp.py", line 94, in drain_events
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit     return connection.drain_events(**kwargs)
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 299, in drain_events
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit     chanmap, None, timeout=timeout,
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 362, in _wait_multiple
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit     channel, method_sig, args, content = read_timeout(timeout)
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 326, in read_timeout
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit     return self.method_reader.read_method()
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit   File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 189, in read_method
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit     raise m
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit IOError: Socket closed
2014-11-28 23:33:50.956 7214 TRACE oslo.messaging._drivers.impl_rabbit 
2014-11-28 23:33:50.957 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:51.954 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:51.977 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:52.004 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:52.024 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:52.043 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:56.977 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] AMQP server on 10.0.1.111:5672 is unreachable: timed out. Trying again in 1 seconds.
2014-11-28 23:33:57.004 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] AMQP server on 10.0.1.111:5672 is unreachable: timed out. Trying again in 1 seconds.
2014-11-28 23:33:57.024 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] AMQP server on 10.0.1.111:5672 is unreachable: timed out. Trying again in 1 seconds.
2014-11-28 23:33:57.043 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] AMQP server on 10.0.1.111:5672 is unreachable: timed out. Trying again in 1 seconds.
2014-11-28 23:33:57.061 7214 ERROR oslo.messaging._drivers.impl_rabbit [-] AMQP server on 10.0.1.111:5672 is unreachable: timed out. Trying again in 1 seconds.
2014-11-28 23:33:57.979 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:58.005 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:58.025 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:58.043 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:58.062 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Delaying reconnect for 1.0 seconds...
2014-11-28 23:33:58.979 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:59.005 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:59.025 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:59.044 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:33:59.062 7214 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:34:00.904 7214 ERROR neutron.agent.linux.utils [req-c4fee448-5fbe-4b0e-84ea-db4bb635734b None] 
Command: ['ps', '--ppid', '7303', '-o', 'pid=']
Exit code: 1
Stdout: ''
Stderr: ''
2014-11-28 23:38:04.875 2193 INFO neutron.common.config [-] Logging enabled!
2014-11-28 23:38:05.995 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:06.021 2193 ERROR oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] AMQP server on 10.0.1.111:5672 is unreachable: [Errno 111] ECONNREFUSED. Trying again in 1 seconds.
2014-11-28 23:38:07.023 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Delaying reconnect for 1.0 seconds...
2014-11-28 23:38:08.024 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:08.037 2193 ERROR oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] AMQP server on 10.0.1.111:5672 is unreachable: [Errno 111] ECONNREFUSED. Trying again in 3 seconds.
2014-11-28 23:38:11.038 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Delaying reconnect for 1.0 seconds...
2014-11-28 23:38:12.040 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:12.052 2193 ERROR oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] AMQP server on 10.0.1.111:5672 is unreachable: [Errno 111] ECONNREFUSED. Trying again in 5 seconds.
2014-11-28 23:38:17.055 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Delaying reconnect for 1.0 seconds...
2014-11-28 23:38:18.057 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.068 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.079 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.091 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.101 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.111 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.120 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.130 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.138 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.148 2193 INFO oslo.messaging._drivers.impl_rabbit [req-46531ce0-b1f7-4608-ac33-4f5738eb402b ] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.167 2193 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.184 2193 INFO oslo.messaging._drivers.impl_rabbit [-] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.191 2193 INFO oslo.messaging._drivers.impl_rabbit [-] Connecting to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.208 2193 INFO oslo.messaging._drivers.impl_rabbit [-] Connected to AMQP server on 10.0.1.111:5672
2014-11-28 23:38:18.238 2193 INFO neutron.plugins.openvswitch.agent.ovs_neutron_agent [req-46531ce0-b1f7-4608-ac33-4f5738eb402b None] Mapping physical network physnet-tenant to bridge br-tenant
2014-11-28 23:38:19.322 2193 INFO neutron.plugins.openvswitch.agent.ovs_neutron_agent [req-46531ce0-b1f7-4608-ac33-4f5738eb402b None] Agent initialized successfully, now running... 
2014-11-28 23:38:19.331 2193 INFO neutron.plugins.openvswitch.agent.ovs_neutron_agent [req-46531ce0-b1f7-4608-ac33-4f5738eb402b None] Agent out of sync with plugin!
2014-11-28 23:39:18.214 2193 ERROR neutron.plugins.openvswitch.agent.ovs_neutron_agent [-] Failed reporting state!
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent Traceback (most recent call last):
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/neutron/plugins/openvswitch/agent/ovs_neutron_agent.py", line 259, in _report_state
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     self.use_call)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/neutron/agent/rpc.py", line 70, in report_state
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     return self.call(context, msg)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/neutron/common/log.py", line 34, in wrapper
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     return method(*args, **kwargs)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/neutron/common/rpc.py", line 161, in call
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     context, msg, rpc_method='call', **kwargs)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/neutron/common/rpc.py", line 187, in __call_rpc_method
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     return func(context, msg['method'], **msg['args'])
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/oslo/messaging/rpc/client.py", line 389, in call
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     return self.prepare().call(ctxt, method, **kwargs)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/oslo/messaging/rpc/client.py", line 152, in call
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     retry=self.retry)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/oslo/messaging/transport.py", line 90, in _send
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     timeout=timeout, retry=retry)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/amqpdriver.py", line 408, in send
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     retry=retry)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/amqpdriver.py", line 397, in _send
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     result = self._waiter.wait(msg_id, timeout)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/amqpdriver.py", line 285, in wait
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     reply, ending = self._poll_connection(msg_id, timeout)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent   File "/usr/lib/python2.7/site-packages/oslo/messaging/_drivers/amqpdriver.py", line 235, in _poll_connection
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent     % msg_id)
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent MessagingTimeout: Timed out waiting for a reply to message ID ab28ee5b9a1243c48e2fe90c3ea8f0d4
2014-11-28 23:39:18.214 2193 TRACE neutron.plugins.openvswitch.agent.ovs_neutron_agent 
2014-11-28 23:39:18.218 2193 WARNING neutron.openstack.common.loopingcall [-] task run outlasted interval by 30.052651 sec
